### DISCUSSION TITLE
Authentification error message

### DISCUSSION THREAD

**Discussion by Zipporah1997** [community/community#48873] [Repositories]
Good day, each time I try to push my code on git I keep getting a message that says access denied to my repo and so I can't push my code.

**Comment by yasgo** [comment]
Hi,

There could be multiple reasons for this.

- If you are using GitHub Desktop, you may not be logged in. [More](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/installing-and-authenticating-to-github-desktop/authenticating-to-github)
- You may need to configure SSH settings. [More](https://docs.github.com/en/authentication/connecting-to-github-with-ssh)

Hope these checks help.
